src/Illuminate/Foundation/Testing/CrawlerTrait.php

```diff
@@ -93,6 +93,26 @@ public function post($uri, array $data = [], array $headers = [])
     }
 
     /**
+     * Visit the given URI with a POST request with content type of application/json.
+     *
+     * @param string $uri
```

Comment:
> I was just following the style in the rest of the file.

Look closer at the positioning of spaces, and you'll see you've not matched the rest of the file.